wallet/src/controller.rs

```diff
@@ -679,7 +679,21 @@ where
 		))
 	}
 
-	fn handle_request(&self, req: Request<Body>) -> WalletResponseFuture {
+	fn handle_get_request(&self, req: Request<Body>) -> WalletResponseFuture {
+		match req
```

Comment:
We added some helpers to `api` crate, perhaps you may want to use it,  eg this macro does the same thing but safer, without calling `unwrap` https://github.com/mimblewimble/grin/blob/master/api/src/web.rs#L143 an example: https://github.com/mimblewimble/grin/blob/master/api/src/handlers/blocks_api.rs#L70